windows_docker_resources/Dockerfile

```diff
@@ -18,53 +18,50 @@ ARG WINDOWS_RELEASE_VERSION=$WINDOWS_RELEASE_ID
 
-# Install Chocolatey by powershell script
+# Install MSVC 2019
+RUN powershell -noexit irm https://aka.ms/vs/16/release/vs_buildtools.exe -OutFile vs_buildtools_2019.exe
+RUN vs_buildtools_2019.exe --quiet --wait --norestart --add Microsoft.Component.MSBuild --add Microsoft.Net.Component.4.6.1.TargetingPack --add Microsoft.Net.Component.4.8.SDK --add Microsoft.VisualStudio.Component.CoreBuildTools --add Microsoft.VisualStudio.Component.Roslyn.Compiler --add Microsoft.VisualStudio.Component.TextTemplating --add Microsoft.VisualStudio.Component.VC.CLI.Support --add Microsoft.VisualStudio.Component.VC.CoreBuildTools --add Microsoft.VisualStudio.Component.VC.CoreIde --add Microsoft.VisualStudio.Component.VC.Redist.14.Latest --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK --add Microsoft.VisualStudio.Component.Windows10SDK.19041 --add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core --add Microsoft.VisualStudio.Workload.MSBuildTools --add Microsoft.VisualStudio.Workload.VCTools
 
-# Pinning chocolatey version to 1.4.0 previous one to major bump to 2.0.0 due to issues with chocolatey_package chef resource
-# See https://github.com/chef/chef/issues/13751 for more detail of the issue
-# This should be solved for chef version 18; see https://github.com/chef/chef/pull/13833
-RUN powershell -noexit "$env:chocolateyDownloadUrl = 'https://community.chocolatey.org/api/v2/package/chocolatey/1.4.0'; Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
+# Install MSVC 2022
+RUN powershell -noexit irm https://aka.ms/vs/17/release/vs_buildtools.exe -OutFile vs_buildtools_2022.exe
+RUN vs_buildtools_2022.exe --quiet --wait --norestart --add Microsoft.Component.MSBuild --add Microsoft.Net.Component.4.6.1.TargetingPack --add Microsoft.Net.Component.4.8.SDK --add Microsoft.VisualStudio.Component.CoreBuildTools --add Microsoft.VisualStudio.Component.Roslyn.Compiler --add Microsoft.VisualStudio.Component.TextTemplating --add Microsoft.VisualStudio.Component.VC.CLI.Support --add Microsoft.VisualStudio.Component.VC.CoreBuildTools --add Microsoft.VisualStudio.Component.VC.CoreIde --add Microsoft.VisualStudio.Component.VC.Redist.14.Latest --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK --add Microsoft.VisualStudio.Component.Windows10SDK.19041 --add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core --add Microsoft.VisualStudio.Workload.MSBuildTools --add Microsoft.VisualStudio.Workload.VCTools
 
-# choco installs. chef-workstation is being installed to get berks and download cookbook dependencies
-RUN choco install -y git chef-workstation
+# Install pixi
+RUN powershell -noexit "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://pixi.sh/install.ps1'))"
```

Comment:
This was true with omnitruck when installing cinc.sh, but noting that this is a curl|bash equivalent which if pixi's infrastructure is compromised could lead to a compromise of our CI container.

Reply:
Yeah, it isn't my favorite.  A couple of alternative ideas:

1.  Bake our own Windows docker image that already has this in it, pinned to a particular version that was hand-verified by a human.  This has the advantage of pinning it, and making sure a compromised upstream doesn't compromise us, but the downside is that we'll have to refresh the image by hand.
2.  Download a copy of the pixi installer, have a human verify it, and host it locally on ROS infrastructure.  Then have the Dockerfile reference that.  This is slightly better than the above in that it is easier to update if needed, but then we need a place to put the pixi installer.  Also, if the thing that install.ps1 points to is compromised, this wouldn't necessarily help.

Thoughts?

Reply:
In release-tools for the build.o.o Jenkins, I'm downloading the pixi.exe binary directly from github:
```
 https://github.com/prefix-dev/pixi/releases/download/v%PIXI_VERSION%/pixi-x86_64-pc-windows-msvc.exe 
```
Since it is a pinning version we can add a checksum check before using the binary.

Reply:
So I did get this to work on another branch.  However, in the spirit of that last comment, I'm going to leave this PR as-is with the `install.ps1` file for now.  Once we merge this in, I'll open up a separate PR which switches to the pinned version (it was a little more difficult than I anticipated).